### DISCUSSION TITLE
Anyone working on this project?

### DISCUSSION THREAD

**Discussion by jrglynn2** [semaphoreui/semaphore#1347] [General]
Is there actually anyone working on this project? I entered a couple of feature requests (issues section) and discussion posts ~6 weeks ago, and not gotten any sort of feedback at all....

Thanks!

**Comment by ansibleguy** [comment]
Greetings!

It seems the current state is a combination of the main operator [not having enough time](https://github.com/ansible-semaphore/semaphore/pull/1237#issuecomment-1627449858), the project [not having enough backers, the lack of 'teamwork'](https://github.com/ansible-semaphore/semaphore/discussions/1111) and a flood of issues/feature requests.

I too am a little sad as this project has so much potential as an Ansible AWX alternative!
And as it is implementable as a single container, it could be a plug-and-play building-block of other management solutions.

Edit: I personally would love to contribute to the project as I have programming experience and know the ins-and-outs of Ansible. But it's a little hard getting started with PR's as the structure of the codebase is not documented (from what I've seen). Per example: How the NPM-JS Web-Frontend interacts with the database and the GO-Backend.

\- AnsibleGuy

**Comment by tboerger** [comment]
I think you can see on the activity of this project that it's being worked on to get various issues fixed. There are still not enough people involved, but I think it's improving.
